gradle.properties

```diff
@@ -2,11 +2,19 @@
 version = 0.28.0-SNAPSHOT
 
 # Need increased heap for running Gradle itself, or SonarQube will run the JVM out of metaspace
-org.gradle.jvmargs=-Xmx2048m
+org.gradle.jvmargs = -Xmx2048m
 
 # Configuration for SonarQube checks
-systemProp.sonar.host.url=https://sonarcloud.io
-systemProp.sonar.organization=hashgraph
+systemProp.sonar.host.url = https://sonarcloud.io
+systemProp.sonar.organization = hashgraph
+systemProp.sonar.projectKey = com.hedera.hashgraph:hedera-services
+systemProp.sonar.projectName = Hedera Services
+systemProp.sonar.projectVersion = 0.27.0-SNAPSHOT
```

Comment:
Is there any way we can pick up the project.version instead of restating the version here?

Reply:
Yeah, these shouldn't be in the properties file anyway. Moved to the standard conventions definition. These shouldn't be changed anyway.